core/src/main/java/com/linecorp/armeria/server/DefaultPathMappingContext.java

```diff
@@ -57,20 +57,23 @@
     private final MediaType consumeType;
     @Nullable
     private final List<MediaType> produceTypes;
+    private final boolean isPreflight;
     private final List<Object> summary;
     @Nullable
     private Throwable delayedCause;
 
     DefaultPathMappingContext(VirtualHost virtualHost, String hostname,
                               HttpMethod method, String path, @Nullable String query,
-                              @Nullable MediaType consumeType, @Nullable List<MediaType> produceTypes) {
+                              @Nullable MediaType consumeType, @Nullable List<MediaType> produceTypes,
+                              boolean isPreflight) {
```

Comment:
- How about `isCorsPreflight`?
- How about adding overloaded constructor where `isCorsPreflight` is always `false`?